dspace-api/src/test/java/org/dspace/discovery/DiscoveryIT.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.discovery;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.builder.CollectionBuilder;
+import org.dspace.builder.CommunityBuilder;
+import org.dspace.builder.WorkspaceItemBuilder;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.content.WorkspaceItem;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.WorkspaceItemService;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * This class will aim to test Discovery related use cases
+ */
+public class DiscoveryIT extends AbstractIntegrationTestWithDatabase {
+
+    protected WorkspaceItemService workspaceItemService = ContentServiceFactory.getInstance().getWorkspaceItemService();
+    protected SearchService searchService = SearchUtils.getSearchService();
+
+
+    Community community;
+    Collection col;
+
+    WorkspaceItem leftIs;
+    WorkspaceItem rightIs;
```

Comment:
this is not used in the test, the leftIs name should be replaced with a more meaningful item

Reply:
We fixed this.